### DISCUSSION TITLE
Financials - is there any bank integration or any way to uplaod statement?

### DISCUSSION THREAD

**Discussion by kernny** [sheltermanager/asm3#1382] [Q&A / Help] [answered]
Just checking if there is an easier way to get our transactions from bank accounts into Shelter Manager?

They're doing it manually at the moment and it is very time consuming and errors can be made. 

Are there any options? Can API be used? 

Thanks!

**Comment by Tetzor** [comment, marked as answer]
Hi,

The usual process is to use the account system to record everything, and then reconcile this against the bank statement when it is received. We do have options to import CSV files from Stripe and PayPal. I have added more info on this in the link below:

https://sheltermanager.com/repo/asm3_help/configuration.html#import-a-paypal-csv-file

If you export your bank account data as a CSV, you can import the incoming payments (Settings->Import a CSV file) by changing the column headers to import fields the system recognises eg. DONATIONAMOUNT, DONATIONTYPE (the full list can be found using the link below). We do have plans for further integration to allow the import of expenses in the future. 

https://sheltermanager.com/repo/asm3_help/csvimportfields.html  

Kind regards,

Jon
